specs/examples/README.md

```diff
@@ -0,0 +1,89 @@
+# Nimble Examples
+
+## Problem Statement
+
+Nimble should have high-level well-maintained examples for our supported frameworks.
+
+This HLD sketches out the organization and requirements of new examples and of the current examples.
+
+## Links To Relevant Work Items and Reference Material
+
+- [#726 Create a "Getting Started" walkthrough for Nimble Angular](https://github.com/ni/nimble/issues/726)
+- [#325 Set up stackblitz or codepen template with Nimble](https://github.com/ni/nimble/issues/325)
+
+## Implementation / Design
+
+### Frameworks to support
+- Vanilla: TypeScript + SCSS
+    - No explicit examples supporting vanilla JavaScript + CSS with local npm builds. Strongly encourage TS + SCSS.
+    - Examples will use [vite](https://vitejs.dev/)
+        - Supports TS + SCSS out of the box
+        - Web Chapter does not have an alternate recommendation yet for Vanilla apps
+        - Vite works on Stackblitz and local npm builds
+            - Parcel [does not work on Stackblitz](https://github.com/parcel-bundler/parcel/issues/7748)
+            - Rollup requires extra config for TS and SCSS
+            - Webpack aligns with the underlying technology of Angular apps but adds lots of complexity to self manage (Angular generally manages for you)
+- CDN: Vanilla JS and CSS
```

Comment:
Note to self, using vite to build a library isn't great as it inlines a bunch of assets into JS: https://github.com/vitejs/vite/issues/4454

It's mainly good for the final app build

Reply:
dart-sass with a custom resolver may be an alternative
can maybe use https://github.com/wessberg/sass-extended-importer